continuous_integration/install.sh

```diff
@@ -81,5 +81,10 @@ else
     exit 1
 fi
 
-pip install coverage coveralls
+pip install coverage
```

Comment:
> we should skip it in the others as it requires a significant time to run

I double-checked and looks like you are right. An additional 30" for coverage gathering on top of the 1'30" for the tests only on my machine.